core/templates/pages/teach-page/teach-page.module.ts

```diff
@@ -31,6 +31,7 @@ import { BrowserModule } from '@angular/platform-browser';
 import { downgradeComponent } from '@angular/upgrade/static';
 import { HttpClientModule } from '@angular/common/http';
 import { HTTP_INTERCEPTORS } from '@angular/common/http';
+import { TeachPageComponent } from './teach-page.component';
```

Comment:
Can we please sort all these imports too in a similar way as we did for the component and spec files?

Reply:
Done, I've left a break between Oppia components/services and constants too